torch/onnx/symbolic_opset9.py

```diff
@@ -114,6 +114,12 @@ def div(g, self, other, *args):
         return _div_rounding_mode(g, self, other, *args)
 
 
+@parse_args("v", "v", "v", "f")
+def addcmul(g, self, tensor1, tensor2, value=1):
```

Comment:
Make sure value is always a float:
```suggestion
def addcmul(g, self, tensor1, tensor2, value=1.):
```